Modix.Services/StackExchange/StackExchangeService.cs

```diff
@@ -23,15 +23,20 @@ public class StackExchangeService
             tags = Uri.EscapeDataString(tags);
             var query = _ApiReferenceUrl += $"&site={site}&tags={tags}&q={phrase}";
 
-            var client = new HttpClient();
+            var client = new HttpClient(new HttpClientHandler
```

Comment:
HttpClient should be declared as a private static member of this class, and not created/disposed every single use.